### PR TITLE
feat: add sentry support on openedx

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR adds support for sentry on Open edX

### CODE REVIEW

File: tutorsentry/patches/openedx-dev-dockerfile-post-python-requirements [hunk 1]
```diff
@@ -0,0 +1,2 @@
+RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared \
+    pip install "sentry-sdk==2.8.0"
```
Comment (MoisesGSalas) [inline tutorsentry/patches/openedx-dev-dockerfile-post-python-requirements:2]: Why not `openedx-dockerfile-post-python-requirements` and avoid two patches?

Comment (Ian2012) [reply tutorsentry/patches/openedx-dev-dockerfile-post-python-requirements:2]: For easier testing of the developments on this plugin

Comment (MoisesGSalas) [reply tutorsentry/patches/openedx-dev-dockerfile-post-python-requirements:2]: I'm mean, doesn't the `openedx-dockerfile-post-python-requirements` guarantees that the plugin is installed in both layers?

Comment (Ian2012) [reply tutorsentry/patches/openedx-dev-dockerfile-post-python-requirements:2]: Yes, it does

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**